provisioner/group_vars/all/all.yml

```diff
@@ -39,3 +39,4 @@ valid_security_console_types:
   - splunk
   - qradar
 ibm_community_grid: true
+create_cluster: false
```

Comment:
@goetzrieger Can you add a new line character here? Build was fine, should work after this.

Reply:
Done